src/web/components/Card/components/UL.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+
+import { verticalDivider } from '../lib/verticalDivider';
+
+const ulStyles = (direction?: 'row' | 'column', padded?: boolean) => css`
+    position: relative;
```

Comment:
Maybe leave a note like you did with the others if this is required for absolute positioned elements 🤷‍♂

Reply:
👍 Done